source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/es-419.json

```diff
@@ -278,6 +278,12 @@
   "users": {
     "noneFound": "No se encontraron usuarios con esos criterios de búsqueda.",
     "title": "Administrar usuarios",
+    "addUser": {
```

Comment:
that is not Spanish!

      "placeholder": "Correo electrónico",
      "button": "Agregar Usuario", //typo
      "modalTitle": "Nuevo usuario",
      "modalAddButton": "Agregar"

In any case, we don't need to add locale in different languages because they will get override by Crowind push

Reply:
crowdin requires all non-english translations be handled in crowdin. 
I don't know what the crowdin URL is for this project, but here is what one looks like for https://emberclear.io : https://crowdin.com/project/emberclear